Selenium.Axe/Selenium.Axe.Test/IntegrationTests.cs

```diff
@@ -22,8 +26,35 @@ namespace Selenium.Axe.Test
     [TestCategory("Integration")]
     public class IntegrationTests
     {
-        private IWebDriver _webDriver;
-        private WebDriverWait _wait;
+        private readonly ThreadLocal<IWebDriver> localDriver = new ThreadLocal<IWebDriver>();
```

Comment:
Using `ThreadLocal`s makes me feel a little uneasy, since it's liable to fall over inconsistently if we were to ever add an `async` test method. Maybe a comment warning about this would be nice?

If you wanted to avoid it instead, you could make `InitDriver` return a wrapper that exposes the webdriver/wait and has a `Dispose` to handle the `Quit`+`Dispose`, so the tests can do `using(var driver = InitDriver(browser)) { /* test */ }`. That feels like overkill for these tests though.

Reply:
Updated to use a concurrent dictionary instead.